CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ RELEASING:
 - Bump spring-boot to 2.7.11 [#1416](https://github.com/GIScience/openrouteservice/pull/1416)
 - Fix the maven-shared-utils CVE [#1414](https://github.com/GIScience/openrouteservice/pull/1414)
 - Exclude snakeyaml [#1414](https://github.com/GIScience/openrouteservice/pull/1414)
+- Upgrade geotools from 28.2 to 29.0 [#1414](https://github.com/GIScience/openrouteservice/pull/1414)
```

Comment:
I believe the corresponding CHANGELOG entry should reference this PR and not #1414, right? The same actually applies to the preceding entry about snakeyaml which I would expect to link against #1418.

Reply:
Sure, what I meant was that the snakeyaml entry links to the wrong PR as well. Don't you want to fix that here too?

Reply:
Good catch too. Fixed